src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -49,5 +62,10 @@ protected override bool CanOffer(SyntaxNode body)
 
             return true;
         }
+        protected override SyntaxNode GetParameterNodeAtIndex(int ordinal, SyntaxNode functionDecleration)
+        {
+            var listOfParameters = InitializeParameterHelpers.GetParameters(functionDecleration);
+            return ordinal > listOfParameters.Count && listOfParameters[ordinal] != null ? null : listOfParameters[ordinal];
```

Comment:
This might crash with "index out of range" exception when executed.

Reply:
Added a check for it just in case.